src/viking.js

```diff
@@ -1,11 +1,97 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength){
+        this.health = health;
+        this.strength = strength;
+        
+    }
+    attack(){
+        return this.strength
+    }
+    receiveDamage(damage){
+        this.health -= damage  
+        
+    }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+        super(name, strength)
```

Comment:
Since you are extending the Soldier class, you should pass "health" and "strength" in the super, instead of the name.
When referencing you only needed to do `this.name = name;` because the other values you are extending from the parent class Soldier.